packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js

```diff
@@ -0,0 +1,54 @@
+import { serverSetToken } from './../lib/authenticate'
```

Comment:
Hey @dylankelly Should we use same name for tests directory across the whole project? At this moment, all are using `test` which is same as vue.js and nuxt.js project. I feel `__tests__` introduce the inconsistency.

Reply:
I would like to change this, i think unit tests should always be colocated with corresponding code. I will change this PR and make another which changes this across the project though.

Reply:
@dylankelly Just check incase you forgot this one...